src/browser/provider/built-in/chrome/build-chrome-args.js

```diff
@@ -0,0 +1,22 @@
+const IN_DOCKER_FLAGS = ['--no-sandbox', '--disable-dev-shm-usage'];
+
+export default function buildChromeArgs (config, cdpPort, platformArgs, profileDir, inDocker) {
```

Comment:
I would prefer if we pack arguments in an object. But it's a suggestion, not a requirement :smile: